0rsk.rb

```diff
@@ -39,12 +39,12 @@
     'sentry' => ''
   }
   config = YAML.safe_load(File.open(File.join(File.dirname(__FILE__), 'config.yml'))) unless ENV['RACK_ENV'] == 'test'
-  if ENV['RACK_ENV'] != 'test'
-    Raven.configure do |c|
-      c.dsn = config['sentry']
-      c.release = Rsk::VERSION
-    end
+
+  Raven.configure do |c|
+    c.dsn = config['sentry']
+    c.release = Rsk::VERSION
   end
+
```

Comment:
@artur-intech we prefer not to use empty lines inside method bodies: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Reply:
@artur-intech the empty line above the `configure` is the line that separates one block from another. Here, we are in the block and breaking it into parts only confuses the reader. If you want to separate Senty config, make a new `configure` block.